helloworld-observ-lib/jsonnetfile.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/grafana/jsonnet-libs.git",
+          "subdir": "common-lib"
+        }
+      },
+      "version": "vzhuravlev/common-lib"
```

Comment:
Needs to be adjusted now common-lib is merged

Reply:
done